pkg/plugins/repo/service.go

```diff
@@ -44,7 +44,7 @@ type ManagerCfg struct {
 func NewManager(cfg ManagerCfg) *Manager {
 	return &Manager{
 		baseURL: cfg.BaseURL,
-		client:  NewClient(cfg.SkipTLSVerify, cfg.GrafanaComAPIToken, cfg.Logger),
+		client:  NewClient(cfg.SkipTLSVerify, cfg.GrafanaComAPIToken, cfg.BaseURL, cfg.Logger),
```

Comment:
`cfg.BaseURL` will be `grafanaComAPIURL + "/plugins"` should we strip the `/plugins` suffix before passing to the client?

Reply:
I think it's fine if we leave it like this :slightly_smiling_face: